samples/package.json

```diff
@@ -18,10 +18,18 @@
     "@google-cloud/kms": "^4.0.0",
     "@google-cloud/precise-date": "^4.0.0",
     "@google-cloud/spanner": "^7.14.0",
-    "yargs": "^17.0.0",
-    "protobufjs": "^7.0.0"
+    "protobufjs": "^7.0.0",
+    "yargs": "^17.0.0"
   },
   "devDependencies": {
+    "@google-cloud/opentelemetry-cloud-trace-exporter": "^2.4.1",
+    "@opentelemetry/api": "^1.9.0",
```

Comment:
`@opentelemetry/api` is already part of dependency in spanner package.

Reply:
Gotcha, thank you @surbhigarg92!